tests/socket/main.cpp

```diff
@@ -8,7 +8,7 @@
 using namespace std::chrono_literals;
 namespace kn = kissnet;
 
-int main()
+int main(void)
```

Comment:
This is C++ not C, `(void)` doesn't actually make sense, as empty param already means "no parameters". We don't need command-line arguments, but I'm not against putting `int argc, char* argv[]` here instead, or leave as-is

Reply:
I got into this habit from users that set:

```sh
CFLAGS: "-Werror=unused-parameter -Werror=return-type -Werror=strict-prototypes"
CPPFLAGS: "-Werror=unused-parameter -Werror=return-type"
```

but yes it's unnecessary for C++.  Thanks!